CONTRIBUTING.md

```diff
@@ -164,21 +474,32 @@ New committers are approved by majority vote of the TSC ([see TSC charter](https
         * Use the imperative mood in the subject line
         * Wrap the body at 72 characters
         * Use the body to explain what and why vs. how
-     
-- Example commit message
-
-```
-Fix OOM caused by foo in bar
-
-Foo was pack ratting ByteBuffers causing OOM.
-
-Cherry-pick of c2bf3b0 (https://github.com/foo/bar/pull/123/commits/c2bf3b025ff456921c4c5ff4bcb814740e823061)
-
-Co-authored-by: Foo Bar <foo@bar.com>
-```
+    * Ensure all code is peer reviewed within your own organization or peers before submitting
+    * Implement and/or address existing feedback before requesting further review
+    * Make a good faith effort to locate example or referential code before requesting someone else direct you towards it
+    * If you see a lack of documentation, create a separate commit with draft documentation to fill the gap
+        * This documentation can be iterated on same as any code commit, demonstrate in real time that you are learning the code section
+    * Implement or modify relevant tests, otherwise provide clear documentation why test updates were not necessary
+    * Tag your PR with affected code areas as best as you can, it’s okay to tag too many, better to cut down irrelevant tags than miss getting input from relevant subject matter experts
 
 - Example PR message
-  - please refer [here](https://github.com/prestodb/presto/blob/master/pull_request_template.md)
+    - please refer [here](https://github.com/prestodb/presto/blob/master/pull_request_template.md)
+
+### What not to do for Pull Requests
+* Submit before getting peer review in your own organization
+* Request review without addressing/implementing previous feedback
+* Asking reviewers to provide examples or code references instead of finding them on your own
+* Protesting lack of documentation for a code section
+    * Review the related code, draft initial documentation as a separate commit
```

Comment:
I don't understand the meaning of the indent for "Review the related code, draft initial documentation as a separate commit". 
Is this another "what not to do", or a recommendation of what you should do instead of "Protesting lack of documentation for a code section"?

Reply:
The indent is a suggestion on what you should do instead, I will add some language around that